oak-lucene/src/main/java/org/apache/jackrabbit/oak/plugins/index/lucene/LuceneDocumentMaker.java

```diff
@@ -123,6 +123,9 @@ protected boolean indexTypedProperty(Document doc, PropertyState property, Strin
             if (includePropertyValue(property, i, pd)){
                 doc.add(f);
                 fieldAdded = true;
+                if (tag == Type.STRING.tag()) {
+                    logLargeStringProperties(property.getName(), property.getValue(Type.STRING, i));
+                }
```

Comment:
I'd prefer if we move this in `FulltextDocumentMaker` itself (comment over there too)

Reply:
Here we are iterating and if we do logging in FulltextDocument. I will have to reiterate again on string array. So leaving it as is.

Reply:
Well, I would still prefer that we refractor a bit and keep logging in oak search. But, if you don't want to do that, then at the very least, we should ensure that we put this log in elasticsearch module as well.